hack/lib/vars.bash

```diff
@@ -20,3 +20,7 @@ readonly SERVING_NAMESPACE="${SERVING_NAMESPACE:-knative-serving}"
 declare -a NAMESPACES
 NAMESPACES=("${SERVING_NAMESPACE}" "${SERVERLESS_NAMESPACE}")
 export NAMESPACES
+readonly RUN_KNATIVE_SERVING_UPGRADE_TESTS="${RUN_KNATIVE_SERVING_UPGRADE_TESTS:-true}"
+readonly RUN_KNATIVE_SERVING_E2E="${RUN_KNATIVE_SERVING_E2E:-true}"
+readonly UPGRADE_SERVERLESS="${UPGRADE_SERVERLESS:-"true"}"
```

Comment:
Does this variable ever differ from `RUN_KNATIVE_SERVING_UPGRADE_TESTS`?

Reply:
Yes, you can run serving upgrade tests and instead of upgrading serverless you can upgrade_cluster